crates/red_knot_python_semantic/src/visibility_constraints.rs

```diff
@@ -329,9 +329,9 @@ impl<'db> VisibilityConstraints<'db> {
                         Truthiness::Ambiguous
                     }
                 }
-                PatternConstraintKind::Singleton(..) | PatternConstraintKind::Unsupported => {
-                    Truthiness::Ambiguous
-                }
+                PatternConstraintKind::Singleton(..)
+                | PatternConstraintKind::Class(..)
```

Comment:
We could sometimes infer a static truthiness here (and also for `Singleton` for that matter), but for now we can consider that a possible future feature, to be driven by actual demand. We don't need to add it as a TODO.